Corona-Warn-App/src/main/java/de/rki/coronawarnapp/http/service/VerificationService.kt

```diff
@@ -17,20 +17,23 @@ interface VerificationService {
     suspend fun getRegistrationToken(
         @Url url: String,
         @Header("cwa-fake") fake: String,
+        @Header("cwa-header-padding") headerPadding: String?,
         @Body requestBody: RegistrationTokenRequest
     ): RegistrationTokenResponse
 
     @POST
     suspend fun getTestResult(
         @Url url: String,
         @Header("cwa-fake") fake: String,
+        @Header("cwa-header-padding") headerPadding: String?,
         @Body request: RegistrationRequest
     ): TestResultResponse
 
     @POST
     suspend fun getTAN(
         @Url url: String,
         @Header("cwa-fake") fake: String,
+        @Header("cwa-header-padding") headerPadding: String?,
```

Comment:
Is there a case where we do not have padding?

Reply:
I think currently you are right. The header can be omitted when the header of a request happens to be exactly as big as we need it.